packages/python/accern_xyme/types.py

```diff
@@ -462,3 +464,12 @@ class DagDef(BaseDagDef):
 DeleteBlobResponse = TypedDict('DeleteBlobResponse', {
     "blobs": List[DeleteBlobStatus],
 })
+BlobDetails = TypedDict('BlobDetails', {
+    "blob_id": str,
+    "blob_uri": str,
+    "fname": str,
+})
+FileMap = TypedDict('FileMap', {
```

Comment:
in TS?

Reply:
These are required mainly for upload functionality which is something we have skipped for typescript. Therefore these 2 types do not exist in TS

Reply:
hmm, okay